composer.json

```diff
@@ -10,20 +10,31 @@
         }
     ],
     "require": {
-        "ext-json": "*"
-    },
-    "autoload": {
-        "classmap": [
-            "src/DrupalFinder.php"
-        ]
+        "composer-runtime-api": "^2.2",
+        "php": ">=8.1"
     },
     "autoload-dev": {
         "psr-4": {
-            "DrupalFinder\\Tests\\": "tests/"
+            "DrupalFinder\\Tests\\": "tests/",
+            "DrupalFinder\\": "src/"
```

Comment:
`"DrupalFinder\\": "src/"` must be registered in "autoload" not "autoload-dev"

Reply:
Fixed